### DISCUSSION TITLE
feature for ploting geometric in plot panel

### DISCUSSION THREAD

**Discussion by lucafei** [foxglove/community#1027] [Q&A]
if foxglove can support plot geometric object in plot panel, it would be nice. For me it will show some constrain area in plot while ploting a speed curve or some other curves. And it also will be great if it can support python script in plot panel to handle data in future, which makes it easy to handle data and visualize data.

**Comment by jtbandes** [comment]
You could try this plotly panel from our community extension registry: https://github.com/mvi-llc/plotly-panel
